e2e/worker/package.json

```diff
@@ -7,6 +7,6 @@
         "@nestjs/common": "^9.0.8",
         "rxjs": "^7.5.6",
         "reflect-metadata": "^0.1.13",
-        "@types/node": "^18.6.3"
+        "@types/node": "18.11.9"
```

Comment:
Are the version changes required for the tests?

Reply:
yes. 18.6.3 yield errors when targeting tsc 4.9